src/data/dictionaries/users.js

```diff
@@ -7,3 +7,13 @@ export const USERS = {
         "repeatPassword": "Password123"
     }
 }
+
+export const MOCKED_USER_PROFILE_RESPONSE = {
```

Comment:
it is not dictionary but fixture

Reply:
move it to fixture folder inside the folder with your test

Reply:
fixed